cosi.proto

```diff
@@ -182,19 +182,15 @@ message ProvisionerCreateBucketRequest {
 }
 
 message ProvisionerCreateBucketResponse {
-    // Intentionally left blank
+    // BucketName returned here is expected to be the globally unique 
+	// identifier for the bucket in the object storage provider 
+	string BucketName = 1;
```

Comment:
do we want BucketName here or BucketID?
I thought the response was going to return the full bucket it.